patches/tock/12-pending-deadlock.patch

```diff
@@ -1,9 +1,9 @@
 diff --git a/capsules/src/usb/usbc_ctap_hid.rs b/capsules/src/usb/usbc_ctap_hid.rs
-index 16b80cb10..2b19519e4 100644
+index 16b80cb10..949388b70 100644
 --- a/capsules/src/usb/usbc_ctap_hid.rs
 +++ b/capsules/src/usb/usbc_ctap_hid.rs
 @@ -120,7 +120,6 @@ struct EndpointState {
-
+ 
```

Comment:
This commit introduced a bunch of spaces for some reason?

Reply:
:shrug: 
Perhaps these came from an IDE.

Removed.